learnEcl/0900-string.md

```diff
@@ -5,7 +5,7 @@ slug: string
 
 # STRING
 
-STRING data type is a sequence of characters, digits, and special characters including space. STRINGs are wrapped in single quotation `''`.
+STRING data type is a sequence of characters, digits, and special characters including space. STRINGs are wrapped in single quotation marks `'This is a string.'`.
```

Comment:
Suggest calling out that STRINGS are for ASCII characters, and providing a "see Unicode and UTF8 datatypes for further information" or something along those lines.